rpm-4.8.0-htonll-fix.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/lib/header.c b/lib/header.c
```

Comment:
I created <https://github.com/cms-externals/rpm> which has all the current patches but this one in the `cms/v4.8.0` branch. Does this one work on slc6?